### ISSUE TITLE
/export <hand,inventory>

### ISSUE THREAD

**Issue by JustBru00** [JustBru00/RenamePlugin#105]
First requested in #86:
1. Add command to export item

* This command will export the item or inventory of the player to a file in /plugins/epicrename/items .
* Possible alternative to upload to pastebin/hastebin/cl1p/etc

Commands:
`<required arg>` `[optional arg]`
`/export <hand || inventory>`

Export format:

```
Items:
  <inventorySlotNumber>:
    material: MATERIAL_NAME
    data: 0
    amount: 1
    display_name: "&f%player_name%'s example item"
    lore:
    - "&f  Lore line 0!"
    - "&7This is lore line 1"
    enchantments:
    - 'enchant1:level'
    - 'enchant2:level'
    hide_attributes: <true/false>
    hide_enchantments: <true/false>
    hide_effects: <true/false>
    hide_unbreakable: <true/false>
```
Export format Example:
Item: http://prntscr.com/ijbqji (pretend this sword is in slot 35 (bottom right slot of inv))

1. Type `/export hand`

```
Items:
  '35':
    material: DIAMOND_SWORD
    data: 0
    amount: 1
    display_name: '           &a&lSWORDS'
    lore:
    - '&f'
    - '   &7&oClick to see all custom &f '
    - '&f &7&oenchantments for &a&oswords&7&o! '
    enchantments:
    - 'DURABILITY:1'
    - 'SHARPNESS:1"
    hide_attributes: true
    hide_enchantments: true
    hide_effects: true
    hide_unbreakable: true
```

**Comment by JustBru00** [comment]
Export item(s) to online site for easy /import

**Comment by JustBru00** [comment]
Also requested by @Coltain13 wayyyyy back: https://www.spigotmc.org/threads/epicrename.51650/page-5#post-1509473

**Comment by JustBru00** [comment]
Need to migrate to a custom server instead of pastebin. Pastebin causes issues.

**Comment by JustBru00** [comment]
Closed by 4d99280a3d6cad8b1eed5c5c61f1cecd67536497
